version.txt

```diff
@@ -1,6 +1,7 @@
 APP_NAME Kodi
 COMPANY_NAME XBMC Foundation
 WEBSITE http://kodi.tv
+MIRRORS http://mirrors.kodi.tv
```

Comment:
I don't like to set this it here.
This file is to control the produced artefacts (exe, deb, dmg, apk) for all platforms, not to control compile settings which do not have any effect on the build output.

Reply:
agreed, this doesn't belong here.
And please keep KODI_MIRROR as alias, if APP_MIRROR is not defined.